chapters/operatorsandexpressions.tex

```diff
@@ -531,10 +531,10 @@ \subsection{Numeric Functions and Conversion Functions}\label{numeric-functions-
   \lstinline!(if leftJustified then "-" else "") + String(minimumLength) + "d"!
 \end{itemize}
 
-Form of the \lstinline!format! string:
-According to ANSI-C the format string specifies one conversion specifier (excluding the leading \%), shall not contain length modifiers, and shall not use `\lstinline!*!' for width and/or precision.
-For all numeric values the format specifiers `\lstinline!f!', `\lstinline!e!', `\lstinline!E!', `\lstinline!g!', `\lstinline!G!' are allowed.
-For integral values it is also allowed to use the `\lstinline!d!', `\lstinline!i!', `\lstinline!o!', `\lstinline!x!', `\lstinline!X!', `\lstinline!u!', and `\lstinline!c!' format specifiers (for non-integral values a tool may round, truncate or use a different format if the integer conversion characters are used).
+The ANSI-C style \lstinline!format! string (which cannot be combined with any of the other named arguments) consists of a single conversion specifier without the leading \%.
```

Comment:
```suggestion
The ANSI-C style \lstinline!format! string (which cannot be combined with any of the other named arguments) contains a single conversion specifier without the leading \%, and optionally also flags, width, and precision specifiers.
```
The intent was to support `format="+4.17g"`. The _one_ conversion specifier is just to clarify that you cannot write `format="g d"` (intended for a real and an integer) or `format=""`.

Reply:
There were more problems: skipping this one.